.github/workflows/release.yaml

```diff
@@ -30,3 +27,42 @@ jobs:
             # generate release name with the tag name that was just pushed
           automatic_release_tag: ${{ steps.vars.outputs.tag }}
           prerelease: false
+  Docker-Build:
+    # We'll only build and push when landing on main, not for PRs
+    if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/main' }}
```

Comment:
I don't think we need this, since we'll never run this in response to a PR, right (only on tag)?